pre_commit_hooks/pretty_format_json.py

```diff
@@ -55,6 +56,14 @@ def parse_topkeys(s):  # type: (str) -> List[str]
     return s.split(',')
 
 
+def get_diff(source, target):
+    source_lines = ''.join(source).split('\n')
+    target_lines = ''.join(target).split('\n')
+    d = difflib.Differ()
+    diff = d.compare(source_lines, target_lines)
```

Comment:
there's a shortcut to this, you can do `''.join(difflib.ndiff(source_lines, target_lines))`